scripts/start_civ.sh

```diff
@@ -301,7 +301,7 @@ function set_pt_pci_vfio() {
 }
 
 function set_pt_usb() {
-    local USB_PCI=$(lspci -D |grep "USB controller" | grep -v "Thunderbolt" | grep -o "....:..:..\..")
+    local USB_PCI=$(lspci -D |grep -m1 "USB controller" | grep -o "....:..:..\..")
```

Comment:
So Thunderbolt comes last?
Is there any strong evidence to prove that host controller comes first, device controller comes second, Thunderbolt comes last?
Is there any other way to differentiate them?

Reply:
There is no way to differentiate between USB host and device controller. The below is shown in lspci of CML U RVP:
00:14.0 USB controller: Intel Corporation Device 02ed
00:14.1 USB controller: Intel Corporation Device 02ee

14.0 is host controller and 14.1 is device controller.
It will be similar in EHL also. I couldn't grab a device and so couldn't paste the EHL lspci output here.

Also, Thunderbolt always comes at the last. It is in CML NUC with PCI ID as 39:00.0. In KBL consumer NUC also it has same PCI ID.

Reply:
OK. Then please add some comments in the function. Thanks.